ConsentViewController/Classes/MessageWebViewController.swift

```diff
@@ -140,7 +140,8 @@ class MessageWebViewController: GDPRMessageViewController, WKUIDelegate, WKNavig
     }
 
     func load(url: URL) {
-        if ConnectivityManager.shared.isConnectedToNetwork() {
+        let connectvityManager = ConnectivityManager()
```

Comment:
What about having the `conectivityManager` as an attribute of `GDPRConsentViewController` class? This way we don't need to instantiate it every time.

Reply:
We are using connectivity manager only once except SourcePointClient class, so instead of creating a global variable I have created a local variable.
If we are using that variable multiple times then we can create global one but in this case I feel we can keep this local one so it will exist only in that method and we don't have to take care outside of method.